packages/browser-destinations/src/destinations/adobe-target/trackEvent/index.ts

```diff
@@ -28,10 +36,9 @@ const action: BrowserActionDefinition<Settings, Adobe, Payload> = {
   },
   perform: (Adobe, event) => {
     // Adobe Target only takes certain event types as valid parameters. We are defaulting to "click".
-    const TARGET_EVENT_TYPE = 'click'
     const event_params = {
       ...event.payload.properties,
-      event_name: event.payload.type
+      event_name: event.payload.event_name
```

Comment:
Do we need the `event.` preface? From other Actions destinations, I thought this would just be `payload.event_name` but could be different for web destinations?

```suggestion
      event_name: payload.event_name
```